README.md

```diff
@@ -285,16 +285,9 @@ There should be a built Debian package (a `.deb` file) in your project directory
 > *My volume on Spotify is 100% and it's still too quiet!*
 
 Have you tried turning the volume up using the command `alsamixer`?
-
-No satisfactory results? Try adding normalisation-pregain to the VOLUME_ARGS
-to higher the initial volume
-
-```
--normalisation-pregain 2
-```
-
-NOTE: dB is a logarithmic scale, so small values have a lot more impact
-than in a linear one.
+If you still don't get satisfactory results, try adding `--normalisation-pregain 2`
+to VOLUME_ARGS in the configuration file to increase the initial volume.
+The supplied value is measured in dB, so increasing it by 20 increases the volume by a factor of 10.
```

Comment:
This is horrible advice.

Especially considering:

The new default normalisation-method is dynamic in libresot >= 0.2, meaning it actually uses a limiter. A normalisation-pregain of 20 would just smash against that limiter and be a distorted mess. Even without a limiter with the basic mode the gain would just be reduced automatically to not clip. So basically all you would be doing is spamming the logs with warnings about how you should reduce the gain.

The max normalisation-pregain will soon be 10.0 for the above reason. Even 10.0 is a crazy basically unrealistically high number. The usable positive range maxes out at maybe 6.0 IMHO.

Reply:
Hm, that advice is already in the README and you included it with #378. I just tried to make it more clear what "logarithmic scale" means. If you think the advice of changing `--normalisation-pregain` should be removed, I'm happy to do that. If you think my explanation (+20 means 10 times louder) implies that one should try increasing the `--normalisation-pregain` by 20 and that's the problem I can also modify it to say:

> The supplied value should be positive to increase the volume. For values larger than 6 you may experience significant distortion.

What do you think?

Reply:
> What do you think?

I don't like the idea of suggesting any number other than maybe noting that anything above 10.0 will fail. As in librespot will exit with an error telling you that anything over 10.0 is an invalid value.

Reply:
My plan eventually is to clean up the readme and move most of it to the wiki. It's a mess.

Reply:
I agree that a wiki would be a good place for the very different kinds of information in the README. For now, I removed the last line (line 290 above). Take a look.